src/Explorer/Controls/Settings/SettingsSubComponents/IndexingPolicyComponent.test.tsx

```diff
@@ -25,7 +25,9 @@ describe("IndexingPolicyComponent", () => {
     },
     onIndexingPolicyDirtyChange: () => {
       return;
-    }
+    },
+    indexTransformationProgress: undefined,
+    refreshIndexTransformationProgress: () => new Promise(jest.fn())
```

Comment:
Jest has promise mocking built in `jest.fn.mockResolvedValue` https://jestjs.io/docs/en/mock-function-api#mockfnmockresolvedvaluevalue